code/modules/lighting/lighting_overlay.dm

```diff
@@ -60,6 +60,14 @@
 		needs_update = 1
 		lighting_update_overlays += src
 
+/atom/movable/lighting_overlay/proc/update_shadowling_overlay(var/turf/T)
+	if(T.shadowling_bright_overlay)
```

Comment:
This won't work if we ever update the lighting system. Won't support making it harder to modify.